recipes/defense-finder/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "defense-finder" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mdmparis/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d4916a23fd459a63ca5b8a660c4f60e2234fc2ce839c2f780a98e416a4dc4f7c
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - macsyfinder >=2.0

```

Comment:
I thought grayskull will remove the deps from host. Besides python and pip everything else can go, isn't it?

Reply:
You are correct, I'm currently working on approaches to include a ~20-30mb database.

But, there are going to be some upstream changes needed to avoid patches. So,  I'm going to mark this one as a draft / work in progress.